views/FAQ.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+      
+    <title>Frequently Asked Questions section - Bootdey.com</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.0/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.0/dist/js/bootstrap.bundle.min.js"></script>
+    <link
+    href="../assets/css/faq.css"
+    rel="stylesheet"
+  />
+  </head>
+  <body>
+    <div class="faq_area section_padding_130" id="faq">
+      <div class="container">
+        <div class="row justify-content-center">
+          <div class="col-12 col-sm-8 col-lg-6">
+             
+            <div
+              class="section_heading text-center wow fadeInUp"
+              data-wow-delay="0.2s"
+              style="
+                visibility: visible;
+                animation-delay: 0.2s;
+                animation-name: fadeInUp;
+              "
+            >
+              <h3><span>Frequently </span> Asked Questions</h3>
+              <p>
+                We need to add some data here. 
+              </p>
+              <div class="line"></div>
+            </div>
+          </div>
+        </div>
+        <div class="row justify-content-center">
+           
+          <div class="col-12 col-sm-10 col-lg-8">
+            <div class="accordion faq-accordian" id="faqAccordion">
+              <div
+                class="card border-0 wow fadeInUp"
+                data-wow-delay="0.2s"
+                style="
+                  visibility: visible;
+                  animation-delay: 0.2s;
+                  animation-name: fadeInUp;
+                "
+              >
+                <div class="card-header" id="headingOne">
+                  <h6
+                    class="mb-0 collapsed"
+                    data-toggle="collapse"
+                    data-target="#collapseOne"
+                    aria-expanded="true"
+                    aria-controls="collapseOne"
+                  >
+                    Question 1?<span
+                      class="lni-chevron-up"
+                    ></span>
+                  </h6>
+                </div>
+                <div
+                  class="collapse"
+                  id="collapseOne"
+                  aria-labelledby="headingOne"
+                  data-parent="#faqAccordion"
+                >
+                  <div class="card-body">
+                    <p>
+                      Lorem ipsum dolor sit amet consectetur adipisicing elit.
```

Comment:
We need to create a task to remove the dummy text and replace with API data.

Reply:
Ok